src/scripts/main.js

```diff
@@ -2,7 +2,32 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+for (const person of people) {
+  // eslint-disable-next-line no-shadow
+  const { name, sex, born, died } = person;
+  const row = document.createElement('tr');
+  const name_ = document.createElement('td');
+  const sex_ = document.createElement('td');
```

Comment:
```suggestion
  const nameCell = document.createElement('td');
  const sexCell = document.createElement('td');
```
Don't name it the same as property. It's a bit confusing